sql/logictest: TestSqlLiteLogic failed

sql/logictest.TestSqlLiteLogic [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_SQLiteLogicTestsBazel/5672149?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_SQLiteLogicTestsBazel/5672149?buildTab=artifacts#/) on master @ [03d5260a467b8640de41ee08054c1ce05f91966c](https://github.com/cockroachdb/cockroach/commits/03d5260a467b8640de41ee08054c1ce05f91966c):


```
=== RUN   TestSqlLiteLogic
    test_log_scope.go:79: test logs captured to: /artifacts/tmp/_tmp/7013a0eb507caf06d50eefd8e168b91f/logTestSqlLiteLogic578733334
    test_log_scope.go:80: use -show-logs to present logs inline
=== CONT  TestSqlLiteLogic
    logic.go:4532: -- test log scope end --
--- FAIL: TestSqlLiteLogic (49101.81s)
=== RUN   TestSqlLiteLogic/local/slt_good_0.test#17
=== PAUSE TestSqlLiteLogic/local/slt_good_0.test#17
=== CONT  TestSqlLiteLogic/local/slt_good_0.test#17
[05:38:55] setting distsql_workmem='27530B';
[05:38:55] rng seed: -5432167146477524195

[05:38:55] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3362/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/commute/10/slt_good_20.test: 9558 statements
[05:38:56] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3362/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby_nosort/10/slt_good_0.test: 7142 statements
[05:38:56] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3362/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/commute/10/slt_good_2.test: 29 statements
[05:38:57] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3362/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/commute/10/slt_good_20.test: 10023 statements
[05:38:57] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3362/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby/1000/slt_good_0.test: 1006 statements
[05:38:57] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3362/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/commute/10/slt_good_20.test with config local: 10032 tests, 0 failures
[05:38:58] --- total progress: 692667 statements
=== CONT  TestSqlLiteLogic/local/slt_good_0.test#17
    logic.go:3263: 
         pq: streamer budget: memory budget exceeded: 12441 bytes requested, 14336 currently allocated, 24192 bytes in budget
[07:20:13] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3362/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby/1000/slt_good_0.test: 8108 statements
[07:20:13] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3362/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby/1000/slt_good_0.test with config local: 8108 tests, 1 failures
    logic.go:4291: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3362/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby/1000/slt_good_0.test:37356: error while processing
    logic.go:4291: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3362/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby/1000/slt_good_0.test:37356: too many errors encountered, skipping the rest of the input
        --- FAIL: TestSqlLiteLogic/local/slt_good_0.test#17 (6078.54s)
=== RUN   TestSqlLiteLogic/local
    --- FAIL: TestSqlLiteLogic/local (0.00s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/sql-queries
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestSqlLiteLogic.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/logictest.TestSqlLiteLogic [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_SQLiteLogicTestsBazel/5685343?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_SQLiteLogicTestsBazel/5685343?buildTab=artifacts#/) on master @ [88789feea153b61b379c0240a07482bdfaf0c932](https://github.com/cockroachdb/cockroach/commits/88789feea153b61b379c0240a07482bdfaf0c932):


```
[11:02:05] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/between/100/slt_good_2.test: 5698 statements
[11:02:06] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby_nosort/1000/slt_good_0.test: 9761 statements
[11:02:07] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/commute/1000/slt_good_1.test: 4211 statements
[11:02:07] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/between/100/slt_good_2.test: 5835 statements
[11:02:08] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby/1000/slt_good_0.test: 4778 statements
[11:02:09] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/commute/1000/slt_good_1.test: 4421 statements
[11:02:09] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/between/100/slt_good_2.test: 5952 statements
[11:02:09] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby_nosort/1000/slt_good_0.test: 9762 statements
[11:02:11] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/commute/1000/slt_good_1.test: 4595 statements
[11:02:11] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby/1000/slt_good_0.test: 4781 statements
[11:02:11] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/between/100/slt_good_2.test: 5988 statements
[11:02:12] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby_nosort/1000/slt_good_0.test: 9764 statements
[11:02:13] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/commute/1000/slt_good_1.test: 4797 statements
[11:02:13] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/between/100/slt_good_2.test: 6055 statements
[11:02:15] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/commute/1000/slt_good_1.test: 4951 statements
[11:02:15] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby_nosort/1000/slt_good_0.test: 9765 statements
[11:02:15] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby/1000/slt_good_0.test: 4783 statements
[11:02:15] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/between/100/slt_good_2.test: 6206 statements
[11:02:17] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/commute/1000/slt_good_1.test: 5213 statements
[11:02:18] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby_nosort/1000/slt_good_0.test: 9766 statements
[11:02:18] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/between/100/slt_good_2.test: 6253 statements
[11:02:19] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby/1000/slt_good_0.test: 4784 statements
[11:02:19] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/commute/1000/slt_good_1.test: 5380 statements
[11:02:20] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/between/100/slt_good_2.test: 6260 statements
[11:02:21] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby_nosort/1000/slt_good_0.test: 9769 statements
[11:02:21] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/commute/1000/slt_good_1.test: 5562 statements
[11:02:22] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/between/100/slt_good_2.test: 6465 statements
[11:02:22] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby/1000/slt_good_0.test: 4785 statements
[11:02:23] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/commute/1000/slt_good_1.test: 5833 statements
[11:02:23] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby_nosort/1000/slt_good_0.test: 9771 statements
[11:02:24] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/between/100/slt_good_2.test: 6714 statements
[11:02:25] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/commute/1000/slt_good_1.test: 6108 statements
[11:02:26] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby/1000/slt_good_0.test: 4786 statements
    logic.go:3559: 
        
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/commute/1000/slt_good_1.test:28156: SELECT pk FROM tab4 WHERE ((((col0 IN (SELECT col3 FROM tab4 WHERE col3 = 7862 OR (col0 > 7104) AND col3 IN (SELECT col0 FROM tab4 WHERE ((col4 < 3774.80))) OR col0 >= 4545 OR (col3 > 1867) AND col1 >= 3341.69 AND col1 <= 8546.42 AND col0 <= 6797 AND col0 = 9644 AND (col3 IN (8505,121,8833,5588,941,148)))))) OR ((col3 >= 7199)) OR col3 < 6877)
        expected success, but found
        (53200) streamer budget: memory budget exceeded: 13449 bytes requested, 12288 currently allocated, 25088 bytes in budget
        resource.go:29: in newMemoryBudgetExceededError()
    logic.go:3263: 
         pq: streamer budget: memory budget exceeded: 13449 bytes requested, 12288 currently allocated, 25088 bytes in budget
[11:02:26] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/commute/1000/slt_good_1.test with config local-spec-planning: 6160 tests, 2 failures
    logic.go:4291: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/commute/1000/slt_good_1.test:28160: error while processing
    logic.go:4291: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/commute/1000/slt_good_1.test:28160: too many errors encountered, skipping the rest of the input
[11:02:26] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/between/100/slt_good_2.test: 6875 statements
[11:02:26] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby_nosort/1000/slt_good_0.test: 9772 statements
        --- FAIL: TestSqlLiteLogic/local-spec-planning/slt_good_1.test#04 (60.65s)
=== RUN   TestSqlLiteLogic/local-spec-planning
    --- FAIL: TestSqlLiteLogic/local-spec-planning (0.00s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestSqlLiteLogic.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/logictest.TestSqlLiteLogic [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_SQLiteLogicTestsBazel/5695604?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_SQLiteLogicTestsBazel/5695604?buildTab=artifacts#/) on master @ [54b4fcaa231abd16260fb09722ed291dbd6a4900](https://github.com/cockroachdb/cockroach/commits/54b4fcaa231abd16260fb09722ed291dbd6a4900):


```
        --- FAIL: TestSqlLiteLogic/local/slt_good_1.test#15 (3141.79s)
=== RUN   TestSqlLiteLogic/local
    --- FAIL: TestSqlLiteLogic/local (0.00s)
=== RUN   TestSqlLiteLogic/local-vec-off/slt_good_0.test#17
=== PAUSE TestSqlLiteLogic/local-vec-off/slt_good_0.test#17
=== CONT  TestSqlLiteLogic/local-vec-off/slt_good_0.test#17
[07:41:47] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/commute/10/slt_good_16.test: 5256 statements
[07:41:47] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby_nosort/10/slt_good_0.test: 6013 statements
[07:41:48] setting distsql_workmem='85241B';
[07:41:48] rng seed: 2432029422809987091

[07:41:48] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/in/100/slt_good_0.test: 7122 statements
[07:41:49] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/commute/10/slt_good_16.test: 5664 statements
[07:41:49] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby_nosort/10/slt_good_0.test: 6286 statements
[07:41:50] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby/1000/slt_good_0.test: 1004 statements
[07:41:50] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/in/100/slt_good_0.test: 7204 statements
[07:41:51] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/commute/10/slt_good_16.test: 6085 statements
[07:41:51] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby_nosort/10/slt_good_0.test: 6991 statements
[07:41:52] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby/1000/slt_good_0.test: 1016 statements
[07:41:52] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/in/100/slt_good_0.test: 7242 statements
[07:41:53] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby_nosort/10/slt_good_0.test: 7503 statements
[07:41:53] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/commute/10/slt_good_16.test: 6498 statements
[07:41:54] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby/1000/slt_good_0.test: 1072 statements
[07:41:54] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/in/100/slt_good_0.test: 7335 statements
[07:41:55] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby_nosort/10/slt_good_0.test: 8034 statements
[07:41:55] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/commute/10/slt_good_16.test: 6642 statements
[07:41:56] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby/1000/slt_good_0.test: 1199 statements
[07:41:56] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/in/100/slt_good_0.test: 7453 statements
[07:41:57] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby_nosort/10/slt_good_0.test: 8600 statements
[07:41:57] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/commute/10/slt_good_16.test: 6988 statements
[07:41:58] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby/1000/slt_good_0.test: 1359 statements
[07:41:58] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/in/100/slt_good_0.test: 7602 statements
[07:41:59] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby_nosort/10/slt_good_0.test: 9137 statements
[07:41:59] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/commute/10/slt_good_16.test: 7277 statements
[07:42:00] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby/1000/slt_good_0.test: 1510 statements
[07:42:00] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/in/100/slt_good_0.test: 7860 statements
[07:42:01] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby_nosort/10/slt_good_0.test: 9505 statements
[07:42:01] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/commute/10/slt_good_16.test: 7944 statements
[07:42:02] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby/1000/slt_good_0.test: 1659 statements
[07:42:02] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/in/100/slt_good_0.test: 8039 statements
[07:42:03] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby_nosort/10/slt_good_0.test with config local-vec-off: 10053 tests, 0 failures
[07:42:03] --- total progress: 3248338 statements
=== CONT  TestSqlLiteLogic/local-vec-off/slt_good_0.test#17
    logic.go:3263: 
         pq: streamer budget: memory budget exceeded: 36406 bytes requested, 57344 currently allocated, 76801 bytes in budget
[07:43:07] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby/1000/slt_good_0.test with config local-vec-off: 6557 tests, 1 failures
    logic.go:4291: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby/1000/slt_good_0.test:29741: error while processing
    logic.go:4291: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby/1000/slt_good_0.test:29741: too many errors encountered, skipping the rest of the input
        --- FAIL: TestSqlLiteLogic/local-vec-off/slt_good_0.test#17 (79.94s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestSqlLiteLogic.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/logictest.TestSqlLiteLogic [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_SQLiteLogicTestsBazel/5701227?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_SQLiteLogicTestsBazel/5701227?buildTab=artifacts#/) on master @ [6374bd83f192bd509e50313acb431690b9c3f8ae](https://github.com/cockroachdb/cockroach/commits/6374bd83f192bd509e50313acb431690b9c3f8ae):


```
[19:24:33] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/delete/1/slt_good_0.test: 2072 statements
[19:24:33] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/delete/10/slt_good_0.test: 6708 statements
[19:24:33] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/delete/10/slt_good_1.test: 10516 statements
[19:24:35] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/delete/1/slt_good_0.test: 2087 statements
[19:24:35] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/delete/10/slt_good_0.test: 6776 statements
[19:24:35] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/delete/10/slt_good_1.test: 10532 statements
[19:24:37] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/delete/1/slt_good_0.test: 2163 statements
[19:24:37] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/delete/10/slt_good_0.test: 6793 statements
[19:24:38] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/delete/10/slt_good_1.test: 10559 statements
[19:24:38] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby/1000/slt_good_0.test: 5264 statements
[19:24:39] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/delete/1/slt_good_0.test: 2177 statements
[19:24:39] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/delete/10/slt_good_0.test: 6818 statements
[19:24:40] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/delete/10/slt_good_1.test: 10613 statements
[19:24:40] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby/1000/slt_good_0.test: 5342 statements
[19:24:41] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/delete/1/slt_good_0.test: 2213 statements
[19:24:41] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/delete/10/slt_good_0.test: 6906 statements
[19:24:42] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/delete/10/slt_good_1.test: 10629 statements
[19:24:42] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby/1000/slt_good_0.test: 5529 statements
[19:24:43] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/delete/1/slt_good_0.test: 2245 statements
[19:24:44] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/delete/10/slt_good_0.test: 6920 statements
[19:24:44] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby/1000/slt_good_0.test: 5628 statements
[19:24:44] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/delete/10/slt_good_1.test: 10699 statements
[19:24:46] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/delete/1/slt_good_0.test: 2258 statements
[19:24:46] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby/1000/slt_good_0.test: 5768 statements
[19:24:46] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/delete/10/slt_good_0.test: 6975 statements
[19:24:46] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/delete/10/slt_good_1.test: 10715 statements
[19:24:48] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/delete/1/slt_good_0.test: 2292 statements
[19:24:48] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/delete/10/slt_good_0.test: 6992 statements
[19:24:48] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby/1000/slt_good_0.test: 5892 statements
[19:24:48] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/delete/10/slt_good_1.test: 10742 statements
[19:24:50] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/delete/1/slt_good_0.test: 2308 statements
[19:24:50] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/delete/10/slt_good_0.test: 7017 statements
[19:24:50] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby/1000/slt_good_0.test: 5909 statements
[19:24:50] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/delete/10/slt_good_1.test: 10756 statements
[19:24:50] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/delete/10/slt_good_1.test with config fakedist-spec-planning: 10774 tests, 0 failures
[19:24:52] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby/1000/slt_good_0.test: 5927 statements
[19:24:52] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/delete/1/slt_good_0.test: 2341 statements
[19:24:52] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/delete/10/slt_good_0.test: 7060 statements
[19:24:52] --- total progress: 12161997 statements
=== CONT  TestSqlLiteLogic/fakedist-spec-planning/slt_good_0.test#17
    logic.go:3263: 
         pq: streamer budget: memory budget exceeded: 30503 bytes requested, 57344 currently allocated, 60032 bytes in budget
[19:27:43] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby/1000/slt_good_0.test with config fakedist-spec-planning: 10050 tests, 1 failures
    logic.go:4291: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby/1000/slt_good_0.test:46826: error while processing
    logic.go:4291: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby/1000/slt_good_0.test:46826: too many errors encountered, skipping the rest of the input
[19:27:43] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby/100/slt_good_3.test: 561 statements
        --- FAIL: TestSqlLiteLogic/fakedist-spec-planning/slt_good_0.test#17 (331.16s)
=== RUN   TestSqlLiteLogic/fakedist-spec-planning
    --- FAIL: TestSqlLiteLogic/fakedist-spec-planning (0.00s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestSqlLiteLogic.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/logictest.TestSqlLiteLogic [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_SQLiteLogicTestsBazel/5706637?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_SQLiteLogicTestsBazel/5706637?buildTab=artifacts#/) on master @ [86e007dcb5cbde3501f138c1d768519db3487857](https://github.com/cockroachdb/cockroach/commits/86e007dcb5cbde3501f138c1d768519db3487857):


```
--- FAIL: TestSqlLiteLogic (56737.33s)
=== RUN   TestSqlLiteLogic/local-vec-off
    --- FAIL: TestSqlLiteLogic/local-vec-off (0.00s)
=== RUN   TestSqlLiteLogic/local-vec-off/slt_good_0.test#17
=== PAUSE TestSqlLiteLogic/local-vec-off/slt_good_0.test#17
=== CONT  TestSqlLiteLogic/local-vec-off/slt_good_0.test#17
[09:37:50] setting distsql_workmem='87666B';
[09:37:50] rng seed: 7353898894494701911

[09:37:51] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3265/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/between/10/slt_good_0.test: 7161 statements
[09:37:51] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3265/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby/10/slt_good_6.test: 5786 statements
[09:37:51] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3265/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby/10/slt_good_7.test: 5586 statements
[09:37:52] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3265/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby/1000/slt_good_0.test: 1005 statements
[09:37:53] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3265/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/between/10/slt_good_0.test: 7347 statements
[09:37:53] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3265/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby/10/slt_good_6.test: 6286 statements
[09:37:53] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3265/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby/10/slt_good_7.test: 5600 statements
[09:37:55] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3265/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby/1000/slt_good_0.test: 1016 statements
[09:37:55] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3265/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/between/10/slt_good_0.test: 7443 statements
[09:37:55] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3265/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby/10/slt_good_6.test: 6886 statements
[09:37:55] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3265/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby/10/slt_good_7.test: 5628 statements
[09:37:57] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3265/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby/1000/slt_good_0.test: 1102 statements
[09:37:57] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3265/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/between/10/slt_good_0.test: 7595 statements
[09:37:57] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3265/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby/10/slt_good_6.test: 7461 statements
[09:37:57] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3265/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby/10/slt_good_7.test: 6121 statements
[09:37:59] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3265/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby/1000/slt_good_0.test: 1247 statements
[09:37:59] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3265/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/between/10/slt_good_0.test: 7675 statements
[09:37:59] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3265/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby/10/slt_good_6.test: 7889 statements
[09:37:59] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3265/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby/10/slt_good_7.test: 6729 statements
[09:38:01] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3265/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby/1000/slt_good_0.test: 1417 statements
[09:38:01] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3265/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/between/10/slt_good_0.test: 7859 statements
[09:38:01] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3265/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby/10/slt_good_6.test: 8407 statements
[09:38:01] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3265/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby/10/slt_good_7.test: 7450 statements
[09:38:03] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3265/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby/1000/slt_good_0.test: 1626 statements
[09:38:03] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3265/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/between/10/slt_good_0.test: 8006 statements
[09:38:03] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3265/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby/10/slt_good_6.test: 9160 statements
[09:38:03] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3265/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby/10/slt_good_7.test: 8175 statements
[09:38:05] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3265/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby/1000/slt_good_0.test: 1744 statements
[09:38:05] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3265/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/between/10/slt_good_0.test: 8156 statements
[09:38:05] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3265/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby/10/slt_good_6.test: 10038 statements
[09:38:05] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3265/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby/10/slt_good_6.test with config local-vec-off: 10048 tests, 0 failures
[09:38:05] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3265/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby/10/slt_good_7.test: 9031 statements
[09:38:06] --- total progress: 3518416 statements
=== CONT  TestSqlLiteLogic/local-vec-off/slt_good_0.test#17
    logic.go:3263: 
         pq: streamer budget: memory budget exceeded: 35664 bytes requested, 57344 currently allocated, 76801 bytes in budget
[09:39:30] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3265/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby/1000/slt_good_0.test with config local-vec-off: 8108 tests, 1 failures
    logic.go:4291: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3265/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby/1000/slt_good_0.test:37356: error while processing
    logic.go:4291: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3265/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/logictest_test_/logictest_test.runfiles/com_github_cockroachdb_cockroach/external/com_github_cockroachdb_sqllogictest/test/index/orderby/1000/slt_good_0.test:37356: too many errors encountered, skipping the rest of the input
        --- FAIL: TestSqlLiteLogic/local-vec-off/slt_good_0.test#17 (99.99s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestSqlLiteLogic.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/ccl/sqlitelogictestccl/tests/3node-tenant/generated_test.go
+++ b/pkg/ccl/sqlitelogictestccl/tests/3node-tenant/generated_test.go
@@ -72,8 +72,6 @@ func runSqliteLogicTest(t *testing.T, file string) {
 	// limit than other logic tests get.
 	serverArgs := logictest.TestServerArgs{
 		MaxSQLMemoryLimit: 512 << 20, // 512 MiB
-		// TODO(yuzefovich): remove this once the flake in #84022 is fixed.
-		DisableWorkmemRandomization: true,
 	}
 	logictest.RunLogicTest(t, serverArgs, configIdx, filepath.Join(sqliteLogicTestDir, file))
 }

--- a/pkg/cmd/generate-logictest/templates.go
+++ b/pkg/cmd/generate-logictest/templates.go
@@ -91,8 +91,6 @@ func runSqliteLogicTest(t *testing.T, file string) {
 	// limit than other logic tests get.
 	serverArgs := logictest.TestServerArgs{
 		MaxSQLMemoryLimit: 512 << 20, // 512 MiB
-		// TODO(yuzefovich): remove this once the flake in #84022 is fixed.
-		DisableWorkmemRandomization: true,
 	}
 	logictest.RunLogicTest(t, serverArgs, configIdx, filepath.Join(sqliteLogicTestDir, file))
 }

--- a/pkg/kv/kvclient/kvstreamer/budget.go
+++ b/pkg/kv/kvclient/kvstreamer/budget.go
@@ -90,10 +90,10 @@ func (b *budget) consume(ctx context.Context, bytes int64, allowDebt bool) error
 func (b *budget) consumeLocked(ctx context.Context, bytes int64, allowDebt bool) error {
 	b.mu.AssertHeld()
 	// If we're asked to not exceed the limit (and the limit is greater than
-	// eight bytes - limits of eight bytes or less are treated as a special case
-	// for "forced disk spilling" scenarios like in logic tests), we have to
-	// check whether we'll stay within the budget.
-	if !allowDebt && b.limitBytes > 8 {
+	// twenty bytes - limits of twenty bytes or less are treated as a special
+	// case for "forced disk spilling" scenarios like in logic tests), we have
+	// to check whether we'll stay within the budget.
+	if !allowDebt && b.limitBytes > 20 {
 		if b.mu.acc.Used()+bytes > b.limitBytes {
 			return errors.Wrap(
 				mon.MemoryResource.NewBudgetExceededError(bytes, b.mu.acc.Used(), b.limitBytes),

--- a/pkg/kv/kvclient/kvstreamer/streamer_test.go
+++ b/pkg/kv/kvclient/kvstreamer/streamer_test.go
@@ -174,7 +174,8 @@ func TestStreamerBudgetErrorInEnqueue(t *testing.T) {
 	acc := rootMemMonitor.MakeBoundAccount()
 	defer acc.Close(ctx)
 
-	getStreamer := func(limitBytes int64) *Streamer {
+	const limitBytes = 30
+	getStreamer := func() *Streamer {
 		acc.Clear(ctx)
 		s := getStreamer(ctx, s, limitBytes, &acc)
 		s.Init(OutOfOrder, Hints{UniqueRequests: true}, 1 /* maxKeysPerRow */, nil /* diskBuffer */)
@@ -182,8 +183,7 @@ func TestStreamerBudgetErrorInEnqueue(t *testing.T) {
 	}
 
 	t.Run("single key exceeds limit", func(t *testing.T) {
-		const limitBytes = 10
-		streamer := getStreamer(limitBytes)
+		streamer := getStreamer()
 		defer streamer.Close(ctx)
 
 		// A single request that exceeds the limit should be allowed.
@@ -193,8 +193,7 @@ func TestStreamerBudgetErrorInEnqueue(t *testing.T) {
 	})
 
 	t.Run("single key exceeds root pool size", func(t *testing.T) {
-		const limitBytes = 10
-		streamer := getStreamer(limitBytes)
+		streamer := getStreamer()
 		defer streamer.Close(ctx)
 
 		// A single request that exceeds the limit as well as the root SQL pool
@@ -205,8 +204,7 @@ func TestStreamerBudgetErrorInEnqueue(t *testing.T) {
 	})
 
 	t.Run("multiple keys exceed limit", func(t *testing.T) {
-		const limitBytes = 10
-		streamer := getStreamer(limitBytes)
+		streamer := getStreamer()
 		defer streamer.Close(ctx)
 
 		// Create two requests which exceed the limit when combined.

--- a/pkg/sql/colfetcher/index_join.go
+++ b/pkg/sql/colfetcher/index_join.go
@@ -496,10 +496,12 @@ func NewColIndexJoin(
 		if spec.MaintainOrdering && diskMonitor == nil {
 			return nil, errors.AssertionFailedf("diskMonitor is nil when ordering needs to be maintained")
 		}
-		// Keep 1/8th of the memory limit for the output batch of the cFetcher,
-		// and we'll give the remaining memory to the streamer budget below.
-		cFetcherMemoryLimit = int64(math.Ceil(float64(totalMemoryLimit) / 8.0))
-		streamerBudgetLimit := 7 * cFetcherMemoryLimit
+		// Keep 1/16th of the memory limit for the output batch of the cFetcher,
+		// another 1/16th of the limit for the input tuples buffered by the index
+		// joiner, and we'll give the remaining memory to the streamer budget
+		// below.
+		cFetcherMemoryLimit = int64(math.Ceil(float64(totalMemoryLimit) / 16.0))
+		streamerBudgetLimit := 14 * cFetcherMemoryLimit
 		kvFetcher = row.NewStreamingKVFetcher(
 			flowCtx.Cfg.DistSender,
 			flowCtx.Stopper(),

--- a/pkg/sql/sqlitelogictest/tests/fakedist-disk/generated_test.go
+++ b/pkg/sql/sqlitelogictest/tests/fakedist-disk/generated_test.go
@@ -71,8 +71,6 @@ func runSqliteLogicTest(t *testing.T, file string) {
 	// limit than other logic tests get.
 	serverArgs := logictest.TestServerArgs{
 		MaxSQLMemoryLimit: 512 << 20, // 512 MiB
-		// TODO(yuzefovich): remove this once the flake in #84022 is fixed.
-		DisableWorkmemRandomization: true,
 	}
 	logictest.RunLogicTest(t, serverArgs, configIdx, filepath.Join(sqliteLogicTestDir, file))
 }

--- a/pkg/sql/sqlitelogictest/tests/fakedist-vec-off/generated_test.go
+++ b/pkg/sql/sqlitelogictest/tests/fakedist-vec-off/generated_test.go
@@ -71,8 +71,6 @@ func runSqliteLogicTest(t *testing.T, file string) {
 	// limit than other logic tests get.
 	serverArgs := logictest.TestServerArgs{
 		MaxSQLMemoryLimit: 512 << 20, // 512 MiB
-		// TODO(yuzefovich): remove this once the flake in #84022 is fixed.
-		DisableWorkmemRandomization: true,
 	}
 	logictest.RunLogicTest(t, serverArgs, configIdx, filepath.Join(sqliteLogicTestDir, file))
 }

--- a/pkg/sql/sqlitelogictest/tests/fakedist/generated_test.go
+++ b/pkg/sql/sqlitelogictest/tests/fakedist/generated_test.go
@@ -71,8 +71,6 @@ func runSqliteLogicTest(t *testing.T, file string) {
 	// limit than other logic tests get.
 	serverArgs := logictest.TestServerArgs{
 		MaxSQLMemoryLimit: 512 << 20, // 512 MiB
-		// TODO(yuzefovich): remove this once the flake in #84022 is fixed.
-		DisableWorkmemRandomization: true,
 	}
 	logictest.RunLogicTest(t, serverArgs, configIdx, filepath.Join(sqliteLogicTestDir, file))
 }

--- a/pkg/sql/sqlitelogictest/tests/local-legacy-schema-changer/generated_test.go
+++ b/pkg/sql/sqlitelogictest/tests/local-legacy-schema-changer/generated_test.go
@@ -71,8 +71,6 @@ func runSqliteLogicTest(t *testing.T, file string) {
 	// limit than other logic tests get.
 	serverArgs := logictest.TestServerArgs{
 		MaxSQLMemoryLimit: 512 << 20, // 512 MiB
-		// TODO(yuzefovich): remove this once the flake in #84022 is fixed.
-		DisableWorkmemRandomization: true,
 	}
 	logictest.RunLogicTest(t, serverArgs, configIdx, filepath.Join(sqliteLogicTestDir, file))
 }

--- a/pkg/sql/sqlitelogictest/tests/local-vec-off/generated_test.go
+++ b/pkg/sql/sqlitelogictest/tests/local-vec-off/generated_test.go
@@ -71,8 +71,6 @@ func runSqliteLogicTest(t *testing.T, file string) {
 	// limit than other logic tests get.
 	serverArgs := logictest.TestServerArgs{
 		MaxSQLMemoryLimit: 512 << 20, // 512 MiB
-		// TODO(yuzefovich): remove this once the flake in #84022 is fixed.
-		DisableWorkmemRandomization: true,
 	}
 	logictest.RunLogicTest(t, serverArgs, configIdx, filepath.Join(sqliteLogicTestDir, file))
 }

--- a/pkg/sql/sqlitelogictest/tests/local/generated_test.go
+++ b/pkg/sql/sqlitelogictest/tests/local/generated_test.go
@@ -71,8 +71,6 @@ func runSqliteLogicTest(t *testing.T, file string) {
 	// limit than other logic tests get.
 	serverArgs := logictest.TestServerArgs{
 		MaxSQLMemoryLimit: 512 << 20, // 512 MiB
-		// TODO(yuzefovich): remove this once the flake in #84022 is fixed.
-		DisableWorkmemRandomization: true,
 	}
 	logictest.RunLogicTest(t, serverArgs, configIdx, filepath.Join(sqliteLogicTestDir, file))
 }

